src/utils/index.ts

```diff
@@ -453,9 +458,9 @@ export async function sleep(ms: number = 0) {
 
 /**
  * Wait until a condition is true
- * @param {function(): Promise<boolean>|function(): boolean} condition wait until this callback function returns true
- * @param {number} [timeOutMs=10000] stop waiting after that many milliseconds, -1 for disable
- * @param {number} [pollingIntervalMs=100] check condition between so many milliseconds
+ * @param condition - wait until this callback function returns true
+ * @param timeOutMs - stop waiting after that many milliseconds, -1 for disable
+ * @param pollingIntervalMs - check condition between so many milliseconds
  */
 export async function until(condition: MaybeAsync<() => boolean>, timeOutMs = 10000, pollingIntervalMs = 100) {
```

Comment:
This method is very similar to `waitForCondition` in `streamr-test-utils` (https://github.com/streamr-dev/streamr-test-utils/blob/b7ba196f95541837deb114ad08ce204c90ced214/src/utils.ts#L61), maybe we could use that?

Reply:
yes, correct, only reservation is `streamr-test-utils` sounds like it shouldn't be used outside test env.

Reply:
Yeah, the utils in `streamr-test-utils` are more generic, but the name doesn't tell that... Should maybe rename the npm package some day.